xla/service/gpu/gpu_fused_mha_runner.h

```diff
@@ -328,20 +265,12 @@ class FusedMultiHeadedAttentionBackwardRunner {
   using Repr = std::variant<
```

Comment:
Comment by @bchetioui 

Also looks like std::optional (maybe doesn't matter).

Reply:
Should probably remove this now since it is not variant any more.